onnx/Cargo.toml

```diff
@@ -31,3 +31,7 @@ mapr = "0.8.0"
 
 [build-dependencies]
 prost-build = "0.10.0"
+
+[features]
+default = []
+getrandom-js = ["tract-onnx-opl/getrandom-js"]
```

Comment:
https://docs.rs/getrandom/latest/getrandom/#indirect-dependencies

They suggest to put it all the time as it has no impact on non-webasm targets (that way, you don't need it on the cargo invocation too...). wdyt ?
getrandom = { version = "0.2", features = ["js"] }

Reply:
i think this section of the getrandom documentation is relevant to people making an app that depends on getrandom indirectly, not for libraries?
if I understand correctly, they expect libraries to not specify any backend, and binaries to specify the backend even if they're not depending on getrandom directly.
I'm still a bit confused though, not sure I understand correctly

Reply:
i think that would mean that people using the `custom` feature in getrandom, with the `wasm32-unknown-emscripten` target will have an error when using tract because we force the `js` backend.
https://docs.rs/getrandom/latest/src/getrandom/lib.rs.html#232
https://docs.rs/getrandom/latest/getrandom/macro.register_custom_getrandom.html#registering-a-custom-getrandom-implementation

Reply:
actually, you're right
by setting the js feature we don't force it according to the getrandom sourcecode

Reply:
Let's ask @bminixhofer what he thinks...

Reply:
I haven't used getrandom, but I would also go for `getrandom = { version = "0.2", features = ["js"] }`.

From the docs:

> This feature has no effect on targets other than wasm32-unknown-unknown.

so should be all good.